src/renderer/services/api/eth2.ts

```diff
@@ -83,12 +81,12 @@ export class Eth2 implements IBeaconAPIClient {
         validatorPubkey: BLSPubkey,
         pocBit: uint8,
         slot: Slot,
-        shard: Shard
+        shard: number,
```

Comment:
lel, leave this, I will correct this in Pr with validator api as this has become CommitteeIndex